guide/creating-your-bot/deleting-commands.md

```diff
@@ -8,7 +8,7 @@ You may have decided that you don't need a command anymore and don't want your u
 
 ## Deleting specific commands
 
-To delete a specific command, you will require its id. Head to **Server Settings -> Integrations -> Bots and Apps** and choose your bot. Then, right click a command and click **Copy ID**.
+To delete a specific command, you will need its id. Head to **Server Settings -> Integrations -> Bots and Apps** and choose your bot. Then, right click a command and click **Copy ID**.
```

Comment:
could you replace `id` with `ID` looks better that way imo

Reply:
No.